### PR TITLE
Delete worker

### PR DESCRIPTION
**Pull request by rybmat**

### CODE REVIEW

File: lib/consul-config/src/main/java/com/wikia/gradle/DefaultConsulKeyValueConfig.java [hunk 1]
```diff
@@ -36,21 +41,21 @@ protected static String getDefaultConfigFolder(String app, String env) {
     }
 
     config = new HashMap<>();
+    getRawConfigValues().map(List::stream).orElse(Stream.<Value>empty())
```
Comment (sqreek) [inline lib/consul-config/src/main/java/com/wikia/gradle/DefaultConsulKeyValueConfig.java]: :+1: but should be in another PR, but its ok

File: service/indexer-base/delete-worker/build.gradle [hunk 2]
```diff
@@ -0,0 +1,39 @@
+apply from: rootProject.file('gradle/dropwizard_service.gradle')
+mainClassName = "com.wikia.services.indexerbase.IndexerBaseApplication"
+
+apply plugin: 'java'
+
+sourceCompatibility = 1.5
```
Comment (sqreek) [inline service/indexer-base/delete-worker/build.gradle]: 1.5?

File: service/indexer-base/src/main/java/com/wikia/services/indexerbase/configuration/IndexerConfiguration.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+/**
+ * Dropwizard configuration class, maps yaml configuration into Application configuration
+ */
+public class IndexerConfiguration extends Configuration {
+
+  @Getter
+  @JsonProperty("rabbit")
+  @NotNull
+  private RabbitConnection rabbitConnection = new RabbitConnection();
```
Comment (sqreek) [inline service/indexer-base/src/main/java/com/wikia/services/indexerbase/configuration/IndexerConfiguration.java]: if it's injected from property, it shouldn't be set by new RabbitConnection();
It can be used if Inject have handle all exceptions, and it will be used in application without configuration (or worst, on posible working default configuration)

File: service/indexer-base/src/main/java/com/wikia/services/indexerbase/guice/IndexerBaseModule.java [hunk 4]
```diff
@@ -0,0 +1,21 @@
+package com.wikia.services.indexerbase.guice;
+
+import com.wikia.rabbit.RabbitConnection;
+import com.wikia.services.indexerbase.configuration.IndexerConfiguration;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.Provides;
+
+
+public class IndexerBaseModule extends AbstractModule {
```
Comment (sqreek) [inline service/indexer-base/src/main/java/com/wikia/services/indexerbase/guice/IndexerBaseModule.java]: I'm confused about Base in name.
If it will be used in other services, it shouldn't be inherited anyway becuse, Guice doesn't recommend inheritence aproach in modules. bind.install is using in this type of cases.
So consider change base part, or remove it.

**Comment by sqreek** [comment]
except minor comments LGTM.

File: service/indexer-base/src/test/java/com/wikia/services/indexerbase/integration/IndexerBaseResourceIntegrationTest.java [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+public class IndexerBaseResourceIntegrationTest {
+
+  @Rule
+  public DropwizardAppRule<IndexerConfiguration> rule = new DropwizardAppRule<>(
+      IndexerBaseApplication.class,
+      resourceFilePath("indexer-base.yaml"), ConfigureRandomPorts.RANDOM_PORTS);
+
+  @Test
+  public void successfulResponse() throws Exception {
+
```
Comment (sqreek) [inline service/indexer-base/src/test/java/com/wikia/services/indexerbase/integration/IndexerBaseResourceIntegrationTest.java]: This test doesn't test anything

File: service/indexer-base/src/main/java/com/wikia/services/indexerbase/guice/IndexerBaseModule.java (hunk 4 above)
Comment (rybmat) [inline service/indexer-base/src/main/java/com/wikia/services/indexerbase/guice/IndexerBaseModule.java]: @idradm ?
